src/simulation/TxGenerator.cpp

```diff
@@ -82,6 +82,16 @@ TxGenerator::updateMinBalance()
     }
 }
 
+bool
+TxGenerator::isLive(LedgerKey const& lk, uint32_t ledgerNum) const
+{
+    LedgerSnapshot lsg(mApp);
+    auto ttlEntryPtr = lsg.load(getTTLKey(lk));
+
+    return ttlEntryPtr &&
+           ttlEntryPtr.current().data.ttl().liveUntilLedgerSeq >= ledgerNum;
```

Comment:
Nit: Use LedgerTypeUtils::isLive instead of >=

Reply:
Done